<launch>

  <!-- <include file="$(find very_simple_robot_simulator)/launch/minimal_simulator.launch" /> -->
  <!-- <include file="$(find very_simple_robot_simulator)/launch/openni_simulator.launch" /> -->
  <!-- <include file="$(find very_simple_robot_simulator)/launch/lidar_simulator.launch" /> -->
  <include file="$(find very_simple_robot_simulator)/launch/run_all_py.launch" />
  <!-- <include file="$(find sound_play)/soundplay_node.launch" /> -->


  <node name="dist_ctrl" pkg="pid" type="controller" ns="robot_dist">
    <param name="Kp" value="0.65" />
    <param name="Ki" value="0.00" />
    <param name="Kd" value="0.0" />
    <param name="upper_limit" value="0.3" />
    <param name="lower_limit" value="-0.3" />
  </node>
  
  <node name="ang_ctrl" pkg="pid" type="controller" ns="robot_ang">
    <param name="Kp" value="0.65" />
    <param name="Ki" value="0.00" />
    <param name="Kd" value="0.00" />
    <param name="upper_limit" value="0.7" />
    <param name="lower_limit" value="-0.7" />
    <param name="angle_error" value="true" />
    <param name="angle_wrap" value="2.0*3.14159265359" />
  </node>
  <node pkg="lab-2" name="dead_reckoning_nav" type="controlled_nav.py" output="screen" />
  <!-- <node pkg="lab-1" name="obstacle_detector" type="obstacle_detector.py"/> -->
  <node pkg="lab-2" name="read_poses" type="read_poses.py" />


</launch>